### ISSUE TITLE
Dependency injection of PersonalDataRepository

### ISSUE THREAD

**Issue by dimitriirtimid** [local-motion/product#217]
Problem is dependency injection of Micronaut icm Axon framework. The PersonalDataRepository
needs to be injected in CommandHandlers and EventHandlers. For now a workaround use a
static class getter is used.

**Comment by dimitriirtimid** [comment]
@AnandAili Did we find a solution for this one?

**Comment by AnandAili** [comment]
@dimitriirtimid No, not yet. some time back I had posted this issue to Micronaut community forum, But I did not get solutions at that time. I will post it again and update the issue soon.
